### ISSUE TITLE
start copying docs from system tables to documentation

### ISSUE THREAD

**Issue by DanRoscigno** [ClickHouse/clickhouse-docs#617]
All of these have description, and where applicable, alias info, embedded in system tables:
- functions
- settings
- table functions
- merge tree settings
- metrics
- profile events
- asynchronous metrics

milovidov-desktop :) SELECT description, alias_to FROM system.functions WHERE name = 'pmod' \G

SELECT
    description,
    alias_to
FROM system.functions
WHERE name = 'pmod'

Query id: 23f752a6-e36f-4921-ade8-5f7f8cde1307

Row 1:
──────
description: 
alias_to:    positiveModulo

1 row in set. Elapsed: 0.002 sec. 

milovidov-desktop :) SELECT description, alias_to FROM system.functions WHERE name = 'positiveModulo' \G

SELECT
    description,
    alias_to
FROM system.functions
WHERE name = 'positiveModulo'

Query id: 24c72b7e-408c-4268-bea4-4054e600435d

Row 1:
──────
description: 
Calculates the remainder when dividing `a` by `b`. Similar to function `modulo` except that `positiveModulo` always return non-negative number.
Returns the difference between `a` and the nearest integer not greater than `a` divisible by `b`.
In other words, the function returning the modulus (modulo) in the terms of Modular Arithmetic.
        
alias_to:    

1 row in set. Elapsed: 0.002 sec.

**Comment by alexey-milovidov** [comment]
@DanRoscigno It's not only about system.functions.

Since recently we have a strict requirement (enforced by the CI system) of having documentation for all new functions, settings, table functions, merge tree settings, metrics, profile events, and asynchronous metrics.

This documentation is embedded into ClickHouse, and, therefore can be used by tools like SQL Console. CC @Zach

Example:

```
milovidov-desktop :) SELECT description, alias_to FROM system.functions WHERE name = 'pmod' \G

SELECT
    description,
    alias_to
FROM system.functions
WHERE name = 'pmod'

Query id: 23f752a6-e36f-4921-ade8-5f7f8cde1307

Row 1:
──────
description: 
alias_to:    positiveModulo

1 row in set. Elapsed: 0.002 sec. 

milovidov-desktop :) SELECT description, alias_to FROM system.functions WHERE name = 'positiveModulo' \G

SELECT
    description,
    alias_to
FROM system.functions
WHERE name = 'positiveModulo'

Query id: 24c72b7e-408c-4268-bea4-4054e600435d

Row 1:
──────
description: 
Calculates the remainder when dividing `a` by `b`. Similar to function `modulo` except that `positiveModulo` always return non-negative number.
Returns the difference between `a` and the nearest integer not greater than `a` divisible by `b`.
In other words, the function returning the modulus (modulo) in the terms of Modular Arithmetic.
        
alias_to:    

1 row in set. Elapsed: 0.002 sec.
``

This is going to be extended to other uniform parts of the codebase: data types, table engines, etc.

We need to copy-paste the documentation to the docs website, and later it can be automated and then back-filled.

**Comment by DanRoscigno** [comment]
Thanks @alexey-milovidov

**Comment by DanRoscigno** [comment]
closed by https://github.com/ClickHouse/ClickHouse/commit/34871e6934e3b92f1ecdae82c50c092c874c15ac
